coala_quickstart/generation/Utilities.py

```diff
@@ -179,8 +184,28 @@ def search_for_orig(decorated, orig_name):
                 return found
 
 
+def get_hashbang(file_path):
+    if not os.path.exists(file_path):
+        return None
```

Comment:
Just a nitpick. Is there any reason why is this being `return None` instead of only `return` like any other `return`s in this function?